_includes/footer.html

```diff
@@ -24,6 +24,12 @@ <h4>&nbsp;</h4>
         <li><a href="{{ '/contact/' | prepend:site.baseurl }}">Contact us</a></li>
       </ul>
     </div>
+    <div class="column d-4-12 m-2-12">
+      <h4>&nbsp;</h4>
+      <ul class="sitemap">
+        <li><a href="https://link.justgiving.com/v1/charity/donate/charityId/3118662?tipScheme=TipJar2.1&isRecurring=true&amount=20.00&reference=givingcheckout_tj21">Donate</a></li>
```

Comment:
Now that we have this url in several places, could we hoist it to a constant somewhere? Perhaps in the config?